Assets/Scripts/LightingManager.cs

```diff
@@ -0,0 +1,86 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[ExecuteAlways]
+public class LightingManager : MonoBehaviour
+{
+    // References
+    [SerializeField] public Light Sun;
+    [SerializeField] public Light Moon;
+    [SerializeField] private LightingPresets Preset;
+
+    // Variables
+    [SerializeField, Range(0, 24)] private float TimeOfDay;
+
+    private void Update()
+    {
+        if (Preset == null)
+        {
+            return;
+        }
+
+        if (Application.isPlaying)
+        {
+            TimeOfDay += Time.deltaTime;
+            TimeOfDay %= 24; // clamp
+            UpdateLighting(TimeOfDay / 24f);
+        }
+        else
+        {
+            UpdateLighting(TimeOfDay / 24f);
+        }
+    }
+
+    private void UpdateLighting(float timePercent)
+    {
+        RenderSettings.ambientLight = Preset.AmbientColor.Evaluate(timePercent);
+        RenderSettings.fogColor = Preset.FogColor.Evaluate(timePercent);
+
+        if (Sun != null)
+        {
+            Sun.color = Preset.SunColor.Evaluate(timePercent);
+            Sun.transform.localRotation = Quaternion.Euler(new Vector3((timePercent * 360f) - 90f, 170f, 0));
+            Sun.intensity = Preset.sunIntensityCurve.Evaluate(timePercent);
+        }
+
+        if (Moon != null)

```

Comment:
Hm, we have the Sun in product scope but if we start doing moon things we're going to need to consult with the new Moon product team